eecs280staff/tutorials
# GitHub fixes

The GitHub tutorial had a major mismatch for EECS 280 students.  It didn't show what to do with the starter files.  I made a copy of the original and modified it.  This is an urgent bug fix.


### Validation
- Make sure there no browser console errors from missing screenshots
- Check link in EECS 280 P2 spec that it points to this REPO, not directly to EECS 485
- Read the changes in commit https://github.com/eecs280staff/tutorials/pull/96/commits/65efe4ea05a897cfc1fa3ca8d2fd5b41c32cb4af
